### DISCUSSION TITLE
Showing PointValue in Paraview

### DISCUSSION THREAD

**Discussion by fervaldies** [idaholab/moose#25893] [Q&A General]
Hello, I was wondering if there is any way to display the PointValue in Paraview to see it at the same time as the mesh. I have a model of a plate and the PointValue represents a thermocouple on the surface of the model and I would like to see it in Paraview.
Thanks in advance.

**Comment by GiudGiud** [comment]
Hello

PointValue will be read as a global variable in Paraview. It will be shown as a unique value over the whole mesh. 
You can definitely plot its time dependence in paraview

Guillaume
